community/people/CGK ; )/index.md

```diff
@@ -0,0 +1,20 @@
+---
+Layout: Person
```

Comment:
I'm not sure if these are case-sensitive or not??  I think they are normally lower case

Reply:
I believe I have made the necessary changes for this.